scripts/lighthouseBudget.js

```diff
@@ -69,9 +69,9 @@ const exitResult = isPassing => {
 // START OF SCRIPT
 
 const budget = {
-  ally: 90,
+  ally: 85,
```

Comment:
I've changed this test to point to our new kyrgyz homepage, and this has caused the score to go ⬇️ 😭

Reply:
@karinathomasbbc what are the failures?

Reply:
I've done some analysis here - feel free to read and comment! https://paper.dropbox.com/doc/Lighthouse-Report-Analysis-for-Kyrgz-FrontPage-vs-HomePage--B4CSApWkrXONELMxEzFJf_2TAg-EpvQ4hWxkewJFE2VdmkyK 

I have attached all of the HTML reports to the doc - you’ll need to download them all to view the details (Dropbox displays raw HTML)

The TLDR is:
- we used to run this against a local version of the igbo homepage with lighthouse v5, and the a11y score was **92**
- Running it with lighthouse v6 for kyrgyz results in an a11y score of **87**
- Running it with lighthouse v10 for kyrgyz the a11y score is **100**

I think it will be too hard / near impossible to recreate the very same test igbo homepage as the one which is currently on local (the test data is years old), which is why I decided to use the kyrgyz homepage, which has all of the features we’re working on, and will eventually be going live.

It is not ideal that the scores are different depending on the version of lighthouse, and that we're stuck on an old version of lighthouse, but this is something we will need to look into at another time.

I think we should be OK to downgrade the lighthouse score on this PR, but we have identified some follow up work for the near future:
- why lighthouse in dev tools on Chrome times out on the new homepage (we discovered this when trying to test this, although there is a [Chrome extension](https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk/related) or a [Pagespeed](https://pagespeed.web.dev/) which work just as well)
- can we / how to upgrade version of lighthouse on PR checks, so that we're using the most up-to-date version
- confirm if we're running lighthouse checks against all page types
  - if not, we should be checking all page types to ensure we have good baseline scores

Reply:
Thanks for the explanation and thorough analysis 🙏 

I'm happy for this to be changed as it does look quite dependant on the version of Lighthouse. We will need to revisit our upgrade path at some stage, so it will likely bring the budget values back up once we do that.